src/linear_algebra/tensor_product.lean

```diff
@@ -280,7 +306,7 @@ add_con.add_con_gen_le $ λ x y hxy, match x, y, hxy with
 | _, _, (eqv.of_add_right m n₁ n₂) := (add_con.ker_rel _).2 $
     by simp_rw [add_monoid_hom.map_add, smul.aux_of, tmul_add]
 | _, _, (eqv.of_smul s m n)        := (add_con.ker_rel _).2 $
-    by rw [smul.aux_of, smul.aux_of, ←smul_assoc, smul_tmul, smul_tmul, smul_assoc]
+    by rw [smul.aux_of, smul.aux_of, ←smul_comm, smul_tmul]
```

Comment:
A puzzle - where is the `compatible_smul` instance argument for smul_tmul coming from?